R/execution.r

```diff
@@ -237,6 +237,11 @@ execute = function(request) {
             if (!plot_builds_upon(last_recorded_plot, plotobj)) {
                 send_plot(last_recorded_plot)
             }
+            # need to be set here to capture the size and have it available
+            # when the plot is sent
+            # 7 is the default in repr...
+            attr(plotobj, '.irkernel_width') <- getOption("repr.plot.width", 7)
+            attr(plotobj, '.irkernel_height') <- getOption("repr.plot.height", 7)
```

Comment:
> also single quotes

damn...

I don't like code crashing when an option is deleted by a user, so I think we should add a default here. Let's add it as a TODO, change repr to export the defaults, wait a few month and then change irkernel to use the defaults.